src/classes/application.js

```diff
@@ -204,6 +213,10 @@ module.exports = class Application extends Base {
       }
     }
 
+    log.trace('Ash server loading models')
+    const modelDir = path.join(process.cwd(), 'app/models')
```

Comment:
This will break if you're not running the app from the root of the project. That can happen if the user is running their app with a process monitor or something.

1/10 For the scope of this PR
8/10 issue in general though I think. Surely there is a more reliable way of determining the project root, yeah?